Eevee.Sleep.Bot/Workers/InGameAnnouncementCrawlingWorker.cs

```diff
@@ -75,14 +75,14 @@ ILogger<InGameAnnouncementCrawlingWorker> logger
                 await SaveDetailsAndHistories(details);
 
                 retryCount = 0;
-            } catch (ContentStructureChangedException e) {
+            } catch (DocumentProcessingException e) {
                 retryCount++;
-                _logger.LogError("Failed to get indexes. Web page structure may have changed. retries: {RetryCount}", retryCount);
-                
+                _logger.LogError("{message} retries: {RetryCount}", e.Message, retryCount);
```

Comment:
Capitalize `retries` so it becomes `Retries`.
Also capitalize `{message}` to `{Message}` for consistency.